docs/source/teams/pluggable_auth.md

```diff
@@ -0,0 +1,668 @@
+(pluggable-auth)=
```

Comment:
Minor suggestion (may ignore): should we rename this file to `.rst`

Reply:
Hmm? Its markdown I don't think renaming it would be a good idea.

Reply:
It appears that all of the other documents are in Restructured Text (`.rst`) format, not Markdown (`.md`).

```
[fiftyone]$ find docs/source/teams -type f -name '*.rst'
docs/source/teams/index.rst
docs/source/teams/teams_app.rst
docs/source/teams/teams_plugins.rst
docs/source/teams/migrations.rst
docs/source/teams/api_connection.rst
docs/source/teams/overview.rst
docs/source/teams/dataset_versioning.rst
docs/source/teams/cloud_media.rst
docs/source/teams/installation.rst
docs/source/teams/management_sdk.rst
docs/source/teams/roles_and_permissions.rst
docs/source/teams/secrets.rst
[fiftyone]$ find docs/source/teams -type f -name '*.md'
docs/source/teams/pluggable_auth.md
```

Reply:
Also sets a precedent that more files can be contributed as `.md`, which would be sad to my OCD as they do render slightly differently.

It feels like we might be close to a major docs overhaul (moving to a new platform) so perhaps the inconsistency won't have much time to take hold.